packages/flutter_adaptive_scaffold/pubspec.yaml

```diff
@@ -14,4 +14,4 @@ dependencies:
 
 dev_dependencies:
   flutter_test:
-    sdk: flutter
+    sdk: flutter
```

Comment:
Missing a newline at the end of the file.

Reply:
thx, fixed